modules/cluster-logging-collector-log-forwarding-supported-plugins-5-2.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
```

Comment:
@anpingli Would you review and comment on the following information for 5.2? We have two separate topics for 5.1 and 5.2:
- [Supported log data output types in OpenShift Logging 5.1](https://deploy-preview-35445--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#cluster-logging-collector-log-forwarding-supported-plugins-5-1_cluster-logging-external)
- [Supported log data output types in OpenShift Logging 5.2](https://deploy-preview-35445--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-external.html#cluster-logging-collector-log-forwarding-supported-plugins-5-2_cluster-logging-external)

- 5.2

-